service/controller/key/aws_control_plane.go

```diff
@@ -35,9 +35,6 @@ func ControlPlaneENIResourceName(id int) string {
 }
 
 func ControlPlaneEtcdNodeName(id int) string {
-	if id == 0 {
-		return "etcd"
-	}
```

Comment:
dropping this means we no longer use ELB  dns (`etcd.clusterid.k8s.basedomain.gigantic.io`) for DNS initial state, but instead node DNS `etcd0.clusterid.k8s.basedomain.gigantic.io`